src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleMethodToViewTransformer.java

```diff
@@ -169,4 +182,19 @@ private SamplePageStreamingView createSamplePageStreamingView(
     builder.nextPageTokenName(methodInfo.responsePageTokenName());
     return builder.build();
   }
+
+  private SampleFieldView createSampleFieldView(
+      FieldInfo field, SampleTypeTable typeTable, boolean escapeReservedParamNames) {
+    String name = field.name();
+    if (escapeReservedParamNames && RESERVED_PARAM_NAMES.contains(name)) {
```

Comment:
I couldn't remember why we handle these manually instead of through the normal symbol table mechanism. ?